styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: pink;
+  text-align: left;
+  font-family: "Lato", sans-serif;
+}
+h2 {
+  font-family: "Poppins", sans-serif;
+  line-height: 25px;
+}
+h3 {
+  font-size: 24px;
+  line-height: 18px;
```

Comment:
Seeing a `line-height` that is less than the `font-size` is always concerning to me -- sometimes it's necessary because a font is sized/spaced weirdly, but here I think you are using it in place of removing/updating the default `margin-bottom` on this element.